src/TikaOnDotnet.TextExtractor/TextExtractionResult.cs

```diff
@@ -29,11 +29,9 @@ public override string ToString()
 			var builder = new StringBuilder("Text:\n" + Text + "MetaData:\n");
 
 			foreach (var keypair in Metadata)
-			{
 				builder.AppendFormat("{0} - {1}\n", keypair.Key, keypair.Value);
-			}
 
 			return builder.ToString();
 		}
-	}
+    }
 }
```

Comment:
I know it seems silly but let's try to keep a blank line at the end of files. It makes Git happier.